NEWS.md

```diff
@@ -2,7 +2,13 @@
 
 ### Miscellaneous
 * Added a template to the `pkgdown` site.
-* Updated package authors.
+* Updated DESCRIPTION file to add new author and fix the package's title.
+* Created GitHub issue template.
+* Added Contribution guidelines, Code of Conduct, and Security information.
+* Updated staged.dependencies file.
+* Updated installation instruction in README.
+* Updated GitHub workflow.
+* Updated `logger` URL in vignette.
```

Comment:
Normally, we add NEWS entries to the PRs that influence/impact the user or if the issue involves a huge internal refactor. Here I believe most of the points above do not concern the user except the README entry. @nikolas-burkoff do you agree with this?

Reply:
I agree. That seems to be the most relevant to user based on our discussion.